test/pylib/cpp/boost/boost_facade.py

```diff
@@ -204,6 +204,7 @@ def get_combined_tests(config: Config):
                 current_suite = line.strip().rstrip('*')
                 suites[mode][current_suite] = []
             else:
-                case_name = line.strip().rstrip('*')
-                suites[mode][current_suite].append(case_name)
+                case_name = line.strip()
+                if not case_name.startswith('_'):
```

Comment:
Why are case names starting with `_` special and shouldn't be added? I guess it's related to boost tests parametrization. Please, explain what happens in more details, e.g. like it was done in bad7407718a20bb28870ad03614026fbf015a936 code comment